normalize_label.go

```diff
@@ -25,19 +25,31 @@ import (
 )
 
 // LabelNamer is a helper struct to build label names.
+// It normalizes OpenTelemetry Protocol (OTLP) attribute names to Prometheus-compliant label names.
```

Comment:
Same comment as before, I don't understand the use of a different term for label name translation:

```suggestion
// It translates OpenTelemetry Protocol (OTLP) attribute names to Prometheus-compliant label names.
```